system/logaccess/Azure/LogAnalytics/CurlClient/AzureLogAnalyticsCurlClient.cpp

```diff
@@ -438,6 +442,20 @@ void AzureLogAnalyticsCurlClient::searchMetaData(StringBuffer & search, const Lo
         {
             search.append("\n| project ");
             selectcols.getString(search, ",");
+
```

Comment:
Do the min and default branches need to reset the value of m_IncludeComponentName?